src/test/java/bio/terra/common/BaseTest.java

```diff
@@ -0,0 +1,135 @@
+          49, 100, 45, 98, 53, 49, 99, 45, 52, 48, 102, 49, 45, 56, 100, 54, 54, 45, 56, 99, 52, 97,
+          52, 101, 102, 50, 48, 50, 50, 48, 46, 116, 97, 114, 46, 103, 122, 63, 103, 101, 110, 101,
+          114, 97, 116, 105, 111, 110, 92, 117, 48, 48, 51, 100, 49, 54, 50, 54, 53, 51, 56, 54, 49,
+          56, 48, 55, 50, 48, 50, 51, 92, 117, 48, 48, 50, 54, 97, 108, 116, 92, 117, 48, 48, 51,
+          100, 109, 101, 100, 105, 97, 34, 44, 34, 109, 101, 116, 97, 103, 101, 110, 101, 114, 97,
+          116, 105, 111, 110, 34, 58, 34, 49, 34, 44, 34, 110, 97, 109, 101, 34, 58, 34, 49, 49, 50,
+          50, 101, 102, 49, 100, 45, 98, 53, 49, 99, 45, 52, 48, 102, 49, 45, 56, 100, 54, 54, 45,
+          56, 99, 52, 97, 52, 101, 102, 50, 48, 50, 50, 48, 46, 116, 97, 114, 46, 103, 122, 34, 44,
+          34, 115, 101, 108, 102, 76, 105, 110, 107, 34, 58, 34, 104, 116, 116, 112, 115, 58, 47,
+          47, 119, 119, 119, 46, 103, 111, 111, 103, 108, 101, 97, 112, 105, 115, 46, 99, 111, 109,
+          47, 115, 116, 111, 114, 97, 103, 101, 47, 118, 49, 47, 98, 47, 116, 101, 114, 114, 97, 45,
+          107, 101, 114, 110, 101, 108, 45, 107, 56, 115, 45, 116, 101, 115, 116, 114, 117, 110,
+          110, 101, 114, 45, 114, 101, 115, 117, 108, 116, 115, 47, 111, 47, 49, 49, 50, 50, 101,
+          102, 49, 100, 45, 98, 53, 49, 99, 45, 52, 48, 102, 49, 45, 56, 100, 54, 54, 45, 56, 99,
+          52, 97, 52, 101, 102, 50, 48, 50, 50, 48, 46, 116, 97, 114, 46, 103, 122, 34, 44, 34, 115,
+          105, 122, 101, 34, 58, 34, 50, 49, 50, 50, 34, 44, 34, 115, 116, 111, 114, 97, 103, 101,
+          67, 108, 97, 115, 115, 34, 58, 34, 83, 84, 65, 78, 68, 65, 82, 68, 34, 44, 34, 116, 105,
+          109, 101, 67, 114, 101, 97, 116, 101, 100, 34, 58, 34, 50, 48, 50, 49, 45, 48, 55, 45, 49,
+          55, 84, 49, 54, 58, 49, 54, 58, 53, 56, 46, 49, 52, 55, 90, 34, 44, 34, 116, 105, 109,
+          101, 83, 116, 111, 114, 97, 103, 101, 67, 108, 97, 115, 115, 85, 112, 100, 97, 116, 101,
+          100, 34, 58, 34, 50, 48, 50, 49, 45, 48, 55, 45, 49, 55, 84, 49, 54, 58, 49, 54, 58, 53,
+          56, 46, 49, 52, 55, 90, 34, 44, 34, 117, 112, 100, 97, 116, 101, 100, 34, 58, 34, 50, 48,
+          50, 49, 45, 48, 55, 45, 49, 55, 84, 49, 54, 58, 49, 54, 58, 53, 56, 46, 49, 52, 55, 90,
+          34, 125
+        };
+  }
+
+  public void assertStorageObjectData(StorageObjectData data) {
+    // Check String deserialization
+    assertEquals("terra-kernel-k8s-testrunner-results", data.getBucket());
+    // Check OffsetDateTime deserialization
+    assertEquals("2021-07-07T22:57:14.257Z", data.getTimeCreated().toString());
+    // Check CustomerEncryption deserialization
+    assertEquals(
+        "6ae5555bcdd2681f1c1e5e5721b6d18a82d0aaada01d1295c22268955e9b79d4753c2f1f5f15c6e1ac80b405e366fda2f4ae26fa7390605c802b8ed7cc787c4ec8458f04c9b07fb65cea1e4344644a33bde9ce28d2eae70ff85cbebc45d6d44c7599cbebfe0b6fed5a1f275968efd29a49aedbd0fd93296f03457ebcc72b2c13",
```

Comment:
Where is this value from? Can it be declared as a constant?

Reply:
This value came from the mock storage object data _MOCK_EVENT_GZIP_ used for this unit test. Since it's specific to this unit test I didn't create a constant for it.